### PR TITLE
Removed ThreadLocal

### PR DESCRIPTION
**Pull request by marcusnaslund**
As per previous discussion.
Peer review @sebastianbaginski @fredrikbryntesson

### CODE REVIEW

File: source/system/Exception.ooc [hunk 1]
```diff
@@ -42,42 +41,41 @@ StackFrame: cover from _StackFrame* {
...
-		if (!this _localFrames hasValue())
+		if (this _localFrames isEmpty)
 			stack = Stack<StackFrame> new()
 		else
-			stack = this _localFrames get()
+			stack = this _localFrames
 		frame := StackFrame new()
 		stack push(frame)
-		this _localFrames set(stack)
+		this _localFrames = stack
```
Comment (marcusnaslund) [inline source/system/Exception.ooc]: Hmm this method looks a bit strange now.

**Comment by marcusnaslund** [comment]
Updated according to my own comment, heh.

**Comment by fredrikbryntesson** [comment]
Looks cool.

**Comment by marcusnaslund** [comment]
This fixes #1341 too, right @sebastianbaginski ?

**Comment by sebastianbaginski** [comment]
Yes.

**Comment by emilwestergren** [comment]
This might solve the current crashes on Nexus 6 as well.

**Pull request merged**